crates/polars-core/src/chunked_array/temporal/conversion.rs

```diff
@@ -3,7 +3,9 @@ use chrono::*;
 
 use crate::prelude::*;
 
-/// Number of seconds in a day
+pub(crate) const NS_IN_DAY: i64 = 86_400_000_000_000;
+pub(crate) const US_IN_DAY: i64 = 86_400_000_000;
+pub(crate) const MS_IN_DAY: i64 = 86_400_000;
 pub(crate) const SECONDS_IN_DAY: i64 = 86_400;
```

Comment:
related, there's already

https://github.com/pola-rs/polars/blob/bd126d55d1e2996eea9dbe4be93c324c15646bd5/crates/polars-arrow/src/temporal_conversions.rs#L15-L30

should these constants be consolidated together somewhere?

Reply:
Probably; do we have a place for such things that each crate can access? (I didn't add these consts, just moved them up to the top of the file :)

Reply:
it might need doing separately to be fair